README.md

```diff
@@ -4,6 +4,54 @@ Project made by EKK Ingarden for [Nasa Space Apps 2024](https://www.spaceappscha
 
 https://github.com/user-attachments/assets/c1c3b7da-9f4d-4ddc-b531-02a951505b03
 
+Note: This project was developed as part of a hackathon. While we strived to deliver a functional and engaging application, the code quality may not meet production standards due to time constraints.
+
+
+# 📚 Overview
+The Landsat Web Tracker bridges the gap between ground-based spectral measurements and Landsat Surface Reflectance (SR) data. By providing users with an intuitive web-based interface, the application makes satellite data more accessible and actionable, fostering environmental awareness and empowering individuals to contribute toward sustainability.
+Our solution allows users to:
+
+- Select a target location and time range.
```

Comment:
This is kinda duplicate of Features section, merge those sections together

Reply:
Its overview features section expanse it

Reply:
Kinda, but I don't see a reason to put shorter and longer versions, just combine them into one. If this section would be a lot shorter okay, but both sections are a bit long.

Reply:
huh did u read them?